### DISCUSSION TITLE
Pet detection

### DISCUSSION THREAD

**Discussion by smtibaa** [espressif/esp-csi#203] [Ideas]
Hello,
esp-csi is a very interesting project for human detection but I believe that the pet detection is a must. 
Do you have any plan to integrate it please?

Regards,
Sabri

**Comment by MacChu0315-Espressif** [comment]
Thank you for your idea!  It's an interesting and practical one.

We are currently trying to optimize the algorithm to improve the effect of motion detection and presence detection.  It is hoped that stronger detection capabilities and stronger hardware (such as C5's 5G band electromagnetic signal) can somehow boost the realization of pet detection.

**Reply by smtibaa** [reply]
Thanks, do you made any progress on this feature ?
